### ISSUE TITLE
Support vertical centering of multi-line for text area (directly as well as in CSS)

### ISSUE THREAD

**Issue by ThomasH99** [codenameone/CodenameOne#3749]
It is possible to vertically center a *single line* text area as described here: https://www.reddit.com/r/cn1/comments/syukb6/how_to_center_text_vertically_in_a_textarea/%EF%BF%BC (although it's a bit cumbersome and not possible in CSS).

A work-around indicated by the CN1 team is to enclose it like this: BoxLayout.encloseYCenter(textArea).

However, this is not documented and not obvious to figure out, so it seems the platform would benefit from better support for this quite common use case. 

Basically, it should work so that a UIID that centers text vertically should 'just work' for both single and multi-line TextAreas.
